README.md

```diff
@@ -81,9 +82,24 @@ let b:ale_fixers = { 'purescript': [ 'purs-tidy' ] }
 let g:ale_fix_on_save = 1
 ```
 
+### Spacemacs
+
+[Spacemacs' Purescript layer](https://github.com/syl20bnr/spacemacs/tree/develop/layers/%2Blang/purescript)
+supports formatting using purs-tidy out of the box.
+
+You can run the formatter manually with either `M-x spacemacs/purescript-format` or with the shortcut `SPC m =`.
+
+To enable automatic formatting of the buffer on save, enable `purescript-fmt-on-save` in your spacemacs config:
+
+```elisp
+  (setq-default dotspacemacs-configuration-layers '(
+    (purescript :variables
+                purescript-fmt-on-save t)))
+```
+
 ### VS Code
 
-#### via [Custom Local Formatters](https://marketplace.visualstudio.com/items?itemName=jkillian.custom-local-formatters) 
+#### via [Custom Local Formatters](https://marketplace.visualstudio.com/items?itemName=jkillian.custom-local-formatters)
```

Comment:
:+1: I would suggest an [EditorConfig](https://editorconfig.org/) file to this project to trim trailing whitespace

Reply:
This was made automatically by my editor. I have removed this change so as not to muddle this PR. We can perhaps include whitespace fixes in a future commit that adds the editorconfig.